lib/import/helpers/brca/providers/rvj/rvj_constants.rb

```diff
@@ -0,0 +1,30 @@
+module Import
+  module Helpers
+    module Brca
+      module Providers
+        module Rvj
+          # Constants used by LondonKgcHandlerColorectal
```

Comment:
@FrancescoNea is this comment correct? The branch name references Bristol

Reply:
Hi @ollietulloch sorry, that comment was somehow inherited from another extractor. Now corrected.